static/app/views/organizationStats/usageChart/utils.tsx

```diff
@@ -49,11 +51,11 @@ export function getDateFromUnixTimestamp(timestamp: number) {
 export function getXAxisDates(
   dateStart: string,
   dateEnd: string,
-  dateUtc: boolean = true,
+  dateUtc: boolean = false,
   interval: IntervalPeriod = '1d'
 ): string[] {
   const range: string[] = [];
-  const start = moment(dateStart).utc().startOf('h');
+  const start = moment(dateStart).startOf('h');
   const end = moment(dateEnd).startOf('h');
```

Comment:
Having the start moved to UTC but not the end was causing the start of the range to be misaligned.